site/name-changes.md

```diff
@@ -0,0 +1,166 @@
+* standardizing the names of the generated yaml files
+* requiring that the custom create and specify a directory that the operators' and domains' generated yaml files will be stored in
+
+This changes how operators and domains are created, and also changes the generated artifacts for operators and domains.
+
+We're not providing an upgrade tool or backwards compatibility with the previous names.  Instead, customers need to use the new script, inputs file and parameter names to recreate their operators and domains.
+
+This document lists the customer visible naming changes.  Also, the WebLogic Server Kubernetes Operator documentation has been updated.
+
+## Customer Visible Files
+
+### Files for Creating and Deleting Operators and Domains
+
+The following files are used to create the operator and to create and delete domains.
+
+| Previous File Name | New File Name |
+| --- | --- |
+| kubernetes/create-domain-job.sh | kubernetes/create-weblogic-domain.sh |
+| kubernetes/create-domain-job-inputs.yaml | kubernetes/create-weblogic-domain-inputs.yaml |
+| kubernetes/create-operator-inputs.yaml | kubernetes/create-weblogic-operator-inputs.yaml |
+| kubernetes/create-weblogic-operator.sh | same |
+| kubernetes/delete-domain.sh | kubernetes/delete-weblogic-operator-resources.sh |
+
+### Generated YAML Files for Operators and Domains
+
+The create scripts generate a number of yaml files that are used to configure the corresponding Kubernetes artifacts for the operator and the domains.
+Normally, customers do not use these yaml files.  However, customers can look at them.  They can also change the operator and domain configuration by editing these files and reapplying them.
+
+#### Directory for the Generated YAML Files
+
+Previously, these files were placed in the kubernetes directory (e.g. kubernetes/weblogic-operator.yaml).  Now, they are placed in per-operator and per-domain directories (since a Kubernetes cluster can have more than one operator and an operator can manage more than one domain).
+
+The customer must create a directory that will parent the per-operator and per-domain directories, and use the -o option to pass the name of that directory to the create script, for example:
+  mkdir /scratch/my-user-projects
+  create-weblogic-operator.sh -o /scratch/my-user-projects
+The pathname can either be a full path name, or a relative path name.  If it's a relative pathname, then it's relative to the directory of the shell invoking the create script.
+
+The per-operator directory name is:
+  <user project dir from -o>/weblogic-operators/<operator namespace from the input yaml file's namespace property>
+
+Similarly, the per-domain directory name is:
+  <user project dir from -o>/weblogic-domains/<domain uid from the input yaml file's domainUid property>
+
+#### What If I Mess Up Creating a Domain or Operator And Want To Do It Again?
+
+* TBD - destroy the operator / domain - I don't think we provide scripts for this yet, but will soon
```

Comment:
We just provide "kubectl delete -f weblogic-operator.yaml", which is sufficient.

Reply:
Are you sure that's sufficient?  What about kubectl delete -f weblogic-operator-security.yaml too?

Reply:
Yes, you're right.